db/seeds.rb

```diff
@@ -14,3 +14,17 @@
 category2.videos << Video.new(title: "Air Force 1", description: "Stunning", small_cover_url: "/tmp/family_guy.jpg", large_cover_url: "/tmp/monk_large.jpg")
 category1.videos << Video.new(title: "Notebook", description: "I cried", small_cover_url: "/tmp/monk.jpg", large_cover_url: "/tmp/monk_large.jpg")
 category1.videos << Video.new(title: "Yes Man", description: "Must see", small_cover_url: "/tmp/south_park.jpg", large_cover_url: "/tmp/monk_large.jpg")
+category2.videos << Video.new(title: "Deep Impact", description: "Wow, a gret movie", small_cover_url: "/tmp/monk.jpg", large_cover_url: "/tmp/monk_large.jpg")
+category2.videos << Video.new(title: "Air Force 1", description: "Stunning", small_cover_url: "/tmp/family_guy.jpg", large_cover_url: "/tmp/monk_large.jpg")
+category1.videos << Video.new(title: "Notebook", description: "I cried", small_cover_url: "/tmp/monk.jpg", large_cover_url: "/tmp/monk_large.jpg")
+category1.videos << Video.new(title: "Yes Man", description: "Must see", small_cover_url: "/tmp/south_park.jpg", large_cover_url: "/tmp/monk_large.jpg")
+category2.videos << Video.new(title: "Deep Impact", description: "Wow, a gret movie", small_cover_url: "/tmp/monk.jpg", large_cover_url: "/tmp/monk_large.jpg")
+category2.videos << Video.new(title: "Air Force 1", description: "Stunning", small_cover_url: "/tmp/family_guy.jpg", large_cover_url: "/tmp/monk_large.jpg")
+category1.videos << Video.new(title: "Notebook", description: "I cried", small_cover_url: "/tmp/monk.jpg", large_cover_url: "/tmp/monk_large.jpg")
+category1.videos << Video.new(title: "Yes Man", description: "Must see", small_cover_url: "/tmp/south_park.jpg", large_cover_url: "/tmp/monk_large.jpg")
+category2.videos << Video.new(title: "Deep Impact", description: "Wow, a gret movie", small_cover_url: "/tmp/monk.jpg", large_cover_url: "/tmp/monk_large.jpg")
+category2.videos << Video.new(title: "Air Force 1", description: "Stunning", small_cover_url: "/tmp/family_guy.jpg", large_cover_url: "/tmp/monk_large.jpg")
+category1.videos << Video.new(title: "Notebook", description: "I cried", small_cover_url: "/tmp/monk.jpg", large_cover_url: "/tmp/monk_large.jpg")
+category1.videos << Video.new(title: "Yes Man", description: "Must see", small_cover_url: "/tmp/south_park.jpg", large_cover_url: "/tmp/monk_large.jpg")
+
+User.create(email: "test@example.com", password: "123", full_name: "David")
```

Comment:
In `seeds.rb`, you should be using `!` version of `create`, because it raises an exception when the record would not be saved to db. Otherwise you get nothing, not even a message.